docs/pages/access-controls/guides/guide-to-cluster-wide-idp-hardening.mdx

```diff
@@ -0,0 +1,660 @@
+---
+title: Guide to hardening against IdP compromises
+description: Implement cluster-wide hardening measures.
+---
+
+Protecting your organization against Identity Provider (IdP) compromises is more critical than ever. 
+This guide aims to help you fortify your identity infrastructure and mitigate the risks associated with IdP weaknesses.
+
+## Understanding IdP compromise
```

Comment:
I would remove the first two headings, since these only contain a single paragraph each.

Reply:
Deleted - (I was trying to highlight those points as they were emphasized in the whitepaper, but I ended up removing a lot of that content.)